resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/k8s/Config.scala

```diff
@@ -316,6 +316,14 @@ private[spark] object Config extends Logging {
       .stringConf
       .createOptional
 
+  val AUTO_STOP_ACTIVE_SPARK_CONTEXTS =
+    ConfigBuilder("spark.kubernetes.submit.autoStopActiveSparkContexts")
```

Comment:
This PR introduces a new config namespace, `spark.kubernetes.submit.*`. Instead of this naming, shall we use the existing `spark.kubernetes.driver.`? If you want to include `submit` somewhere, you can add it to the last part of the config name.

Reply:
This config really used at the spark submit so I think it belongs to `spark.kubernetes.submit`. 

If something else will come up in the future which also has effect on the spark submit and Kubernetes related then using the same reasoning we will never start that namespace. I mean there is always a first one.

Reply:
Okay, I'll leave this to you and other committers.